src/lib/utils/logHandler.js

```diff
@@ -55,6 +55,7 @@ class LogHandler {
       processedMessages = [msg.message];
     } else if (Array.isArray(msg.message)) {
       msg.message.forEach(message => {
+        if(message === Object(message)) return; // test for objects - don't know how to handle these
```

Comment:
Is that to not have a `undefined` when printing objects?
If so, I think we should JSON.strigify them instead (maybe in a try catch in case there is a circular dep

Reply:
I'm not exactly sure where the object originated, but this was not expected input to this method. I've updated it to JSON.stringify anyway.